environment.yml

```diff
@@ -0,0 +1,66 @@
+name: cognoma-cancer-data
+dependencies:
```

Comment:
When `execute.sh` run on osx, yields error (linux vs osx crossplatform issue?):
Error: Packages missing in current osx-64 channels: 
  - fontconfig 2.11.1 6
  - libsodium 1.0.10 0
  - zeromq 4.1.4 0